test/e2e/basic_workflow_test.go

```diff
@@ -20,156 +20,146 @@ import (
-	KnDefaultTestImage string = "gcr.io/knative-samples/helloworld-go"
 )
 
-func Setup(t *testing.T) func(t *testing.T) {
-	e = buildEnv(t)
-	k = kn{t, e.Namespace, Logger{}}
-	CreateTestNamespace(t, e.Namespace)
-	return Teardown
-}
-
-func Teardown(t *testing.T) {
-	DeleteTestNamespace(t, e.Namespace)
-}
-
 func TestBasicWorkflow(t *testing.T) {
 	teardown := Setup(t)
 	defer teardown(t)
 
-	t.Run("returns no service before running tests", func(t *testing.T) {
-		testServiceListEmpty(t, k)
-	})
-
-	t.Run("create hello service and returns no error", func(t *testing.T) {
-		testServiceCreate(t, k, "hello")
-	})
-
-	t.Run("returns valid info about hello service", func(t *testing.T) {
-		testServiceList(t, k, "hello")
-		testServiceDescribe(t, k, "hello")
-	})
-
-	t.Run("update hello service's configuration and returns no error", func(t *testing.T) {
-		testServiceUpdate(t, k, "hello", []string{"--env", "TARGET=kn", "--port", "8888"})
-	})
-
-	t.Run("create another service and returns no error", func(t *testing.T) {
-		testServiceCreate(t, k, "svc2")
-	})
-
-	t.Run("returns a list of revisions associated with hello and svc2 services", func(t *testing.T) {
-		testRevisionListForService(t, k, "hello")
-		testRevisionListForService(t, k, "svc2")
-	})
-
-	t.Run("returns a list of routes associated with hello and svc2 services", func(t *testing.T) {
-		testRouteList(t, k)
-		testRouteListWithArgument(t, k, "hello")
-	})
-
-	t.Run("delete hello and svc2 services and returns no error", func(t *testing.T) {
-		testServiceDelete(t, k, "hello")
-		testServiceDelete(t, k, "svc2")
-	})
-
-	t.Run("returns no service after completing tests", func(t *testing.T) {
-		testServiceListEmpty(t, k)
-	})
+	testServiceListEmpty(t, k)
+	testServiceCreate(t, k, "hello")
+	testServiceList(t, k, "hello")
+	testServiceDescribe(t, k, "hello")
+	testServiceUpdate(t, k, "hello", []string{"--env", "TARGET=kn", "--port", "8888"})
+	testServiceCreate(t, k, "svc2")
+	testRevisionListForService(t, k, "hello")
+	testRevisionListForService(t, k, "svc2")
+	testRouteList(t, k)
+	testRouteListWithArgument(t, k, "hello")
+	testServiceDelete(t, k, "hello")
+	testServiceDelete(t, k, "svc2")
+	testServiceListEmpty(t, k)
 }
 
 // Private test functions
 
 func testServiceListEmpty(t *testing.T, k kn) {
 	out, err := k.RunWithOpts([]string{"service", "list"}, runOpts{NoNamespace: false})
-	assert.NilError(t, err)
+	if err != nil {
+		t.Fatalf(fmt.Sprintf("Error executing 'kn service list' command. Error: %s", err.Error()))
+	}
 
-	assert.Check(t, util.ContainsAll(out, "No resources found."))
+	if !strings.Contains(out, "No resources found.") {
+		t.Fatalf("Expected output 'No resources found.' Instead found:\n%s\n", out)
+	}
 }
 
 func testServiceCreate(t *testing.T, k kn, serviceName string) {
 	out, err := k.RunWithOpts([]string{"service", "create",
 		fmt.Sprintf("%s", serviceName),
 		"--image", KnDefaultTestImage}, runOpts{NoNamespace: false})
-	assert.NilError(t, err)
+	if err != nil {
```

Comment:
Why do you revert all these asserts ?

Reply:
I may have done that unintentionally... Let me double check.

Reply:
See previous. Reverted.